content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,217 @@
+     - :ref:`Yes* <inventory/product_management/manufacture>`
+     - No
+   * - Can be in a kit
+     - Yes
+     - Yes
+     - No
+   * - Placed in package
+     - Yes
+     - :ref:`Yes* <inventory/product_management/package>`
+     - No
+   * - Appears on Inventory report
+     - Yes
+     - No
+     - No
+
+.. _inventory/product_management/on-hand-store:
+
+On-hand quantity
+----------------
+
+A storable product's on-hand and forecasted quantity based on incoming and outgoing orders are
+reflected on the product form, accessed by going to :menuselection:`Inventory app --> Products -->
+Products`.
+
+.. figure:: type/on-hand.png
+   :align: center
+   :alt: Show "On Hand" and "Forecast" smart buttons.
+
+   Current and forecasted quantities displayed in the **On Hand** and **Forecasted** smart buttons
+   on the product form.
+
+.. _inventory/product_management/on-hand-con:
+
+On the other hand, consumable products are regarded as always available, and they cannot be managed
+using reordering rules or lot/serial numbers.
+
+.. _inventory/product_management/transfer-store:
+
+Create transfer
+---------------
+
+*Transfers* are any warehouse operation, such as receipts, internal or batch transfers, or
+deliveries.
+
+When creating a transfer for storable products in the *Inventory* app, transfers modify the on-hand
+quantity at each location. For example, when five units of product is moved from the internal
+location `WH/Stock` to `WH/Packing Zone`, the five units recorded at `WH/Stock` is decreased, and
+five units recorded at `WH/Packing Zone` is increased.
+
+.. _inventory/product_management/transfer-con:
+
+For consumable products, although transfers can be created, the exact quantities at each storage
+location are not tracked.
+
+.. _inventory/product_management/transfer-serv:
+
+Service products cannot be included in transfers, but these products can be `linked to projects and
+tasks for deadline tracking <https://www.youtube.com/watch?v=fix2LGkv13c>`_.
+
+Replenishment
+-------------
+
+The replenishment report, accessed by going to :menuselection:`Inventory app --> Operations -->
+Replenishment` is a list of products that have a negative forecasted quantity and have manual
+reordering rules configured to generate purchase orders to replenish the quantities.
+
+.. seealso::
+   :doc:`../product_replenishment/strategies`
+
+Only storable products appear on the replenishment report.
+
+.. _inventory/product_management/replenish:
+
+Although consumable products do not appear on the replenishment report, the :guilabel:`Replenish`
+button is available on the product form of consumable products in :menuselection:`Inventory app -->
+Products --> Products`.
+
+.. image:: type/replenish.png
+   :align: center
+   :alt: alt text
+
+Since quantities of consumable products, users must manually enter the :guilabel:`Quantity` to order
+in the :guilabel:`Low on stock? Let's replenish` pop-up window.
```

Comment:
I think this sentence needs clarification - I'm not sure what it's trying to say